ci-operator/config/stackrox/scanner/stackrox-scanner-master.yaml

```diff
@@ -130,7 +130,22 @@ tests:
   optional: true
   steps:
     test:
-    - ref: stackrox-scanner
+    - as: scale-tests
+      commands: .openshift-ci/dispatch.sh scale-tests
+      credentials:
+      - mount_path: /tmp/secret/stackrox-scanner-secrets
+        name: stackrox-scanner-secrets
+        namespace: test-credentials
+      dependencies:
+      - env: SCANNER_IMAGE
+        name: scanner
+      - env: SCANNER_DB_IMAGE
+        name: scanner-db
+      from: src
+      resources:
+        requests:
+          cpu: 2000m
+          memory: 4000Mi
```

Comment:
Can you confirm we don't have scale tests on slim because they are essentially the same image?

Reply:
Yeah they are essentially the same, and we want to confirm Scanner does not OOM when presented with such a heavy load